src/components/GoogleCalendarButton.js

```diff
@@ -12,12 +14,52 @@ const useStyles = makeStyles({
 });
 
 export default function GoogleCalendarButton() {
+  const sections = useSelector(sectionsSelector);
   const classes = useStyles();
+  const timeZone = '-06:00';
+  const dateOfTheFirstClass = {
```

Comment:
This is fine for now, but we should brainstorm possibilities for making this dynamic instead of written constants.

Reply:
agreed! Also have to figure out a way to deal with the Daylight saving